src/bwapi/bwresources.py

```diff
@@ -338,23 +338,6 @@ def rename(self, name, new_name):
                     "We cannot support automated renaming of channels at this time."
                 )
 
-    def backfill(self, query_id, backfill_date):
```

Comment:
Presumably we're removing this because backfills are automatic now?  Is the same true for rules, and if so should we remove that endpoint too?

Reply:
Yeah I spoke to Yohan who has been doing some work on queries in BCR and he told me that backfills as a concept don't exist anymore outside of query creation. Attempting to run a backfill on a BCR query returns an error. What we call "backfills" for rules in the SDK are actually bulk actions, so should continue to work fine